spring-integration-core/src/test/java/org/springframework/integration/config/InvalidChannelWithMessageStoreParserTests.java

```diff
@@ -17,22 +17,26 @@
 package org.springframework.integration.config;
 
 import org.hamcrest.Matchers;
+import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
-
 import org.springframework.beans.factory.parsing.BeanDefinitionParsingException;
 import org.springframework.context.support.ClassPathXmlApplicationContext;
 
 /**
  * @author Dave Syer
+ * @deprecated replaced by InvalidQueueChannelParserTests and
+ *             InvalidPriorityChannelParserTests
  */
+@Deprecated
 public class InvalidChannelWithMessageStoreParserTests {
```

Comment:
No, it is an overhead. 
This is just test and it isn't a public API, so if you decided to change its name, just do that!
And I understand your point here and know that it was my advice to collect similar tests together.
LGTM for that part!

If we don't need this test at all just remove.
Since you copy/paste an existing tests to a new one class, pleas, show respect and leave Dave Syer's name there as well :wink: